packages/synapse-react-client/src/components/EvaluationFinder/EvaluationFinder.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react'
+import { useGetEvaluationsInfinite } from '../../synapse-queries/evaluation/useEvaluation'
+import { HelpPopover } from '../HelpPopover/HelpPopover'
+import {
+  Box,
+  Button,
+  FormControl,
+  FormGroup,
+  LinearProgress,
+  Typography,
+} from '@mui/material'
+import { GetEvaluationParameters } from '@sage-bionetworks/synapse-types'
+import { Checkbox } from '../widgets/Checkbox'
+
+export type EvaluationFinderProps = Pick<
+  GetEvaluationParameters,
+  'accessType' | 'activeOnly'
+> & {
+  selectedIds: string[]
+  onChange: (newSelectedIds: string[]) => void
+}
+
+export default function EvaluationFinder(props: EvaluationFinderProps) {
+  const { accessType, activeOnly, selectedIds = [], onChange } = props
+  const [currentPage, setCurrentPage] = useState(0)
+  const { data, isLoading, hasNextPage, fetchNextPage, isFetchingNextPage } =
+    useGetEvaluationsInfinite(
+      {
+        accessType,
+        activeOnly,
+      },
+      { keepPreviousData: true },
+    )
+
+  if (isLoading) {
+    return <LinearProgress />
+  }
+  if (!data?.pages) {
+    return <></>
```

Comment:
No content UI instead?

Reply:
IMO we should propagate an error to an error boundary, which makes this case improbable. If we do that, it seems reasonable to show UI that indicates that an unknown error occurred